docs/administration.md

```diff
@@ -107,17 +107,23 @@ next chapter, but it is not recommended. The same applies to the parameters
 
 ### Configuration in Studio
 
-After you have created your GlobalLink settings at `/Settings/Options/Settings/GlobalLink` 
-and linked them to your site, you need to configure your personal 
-GlobalLink parameters. There can be more settings content items like this with different
-names and values to enable site-specific connections. However, global settings 
-like `dayOffsetForDueDate` have to be defined in `/Settings/Options/Settings/GlobalLink`.
-
-Therefore, you need to add a struct to the GlobalLink settings, named
-`globalLink`. **Make sure to restrict read and write rights of this content to 
-those user groups that actually need access to prevent leaking sensitive information.**
-
-Within that struct the following parameters must/can be specified:
+GlobalLink Settings can be configured globally for all sites or specifically 
+for some site. The Settings can be located in the following folders:
```

Comment:
"or specifically for some site." -> "or specifically for some site. Site-specific settings override global settings except for `dayOffsetForDueDate`."